### ISSUE TITLE
Question: How to use Label3D to label multiple instances?

### ISSUE THREAD

**Issue by kdquine** [diegoaldarondo/Label3D#17]
Hi! We're trying to use label3d to label COM and skeleton points for a couple of animals at the same time; do you have any advice as to how best to do this? We haven't found an option in label3d for multiple instances, but we see support for multiple instances in com and dannce train and prediction...

We're currently thinking our best option might be to just define a skeleton with twice the normal number of points, and then reformat the data-file afterward for use with n_instances=2 in dannce; is this our best option?

We would like to be able to do this whether or not we can tell the two animals apart in every frame, ideally.

Thanks so much!

**Comment by diegoaldarondo** [comment]
Yes, for now this is the best option. Ideally I would like to improve this, but it is not high on my priority list at the moment. 

If you don't care about identity you can set the skeletons to the same colors. If you would also like the keypoints to denote identity, you could make them different colors.

**Comment by kdquine** [comment]
Thanks so much!
